<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>554</width>
    <height>333</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Bloc de notas</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../QtProjects/notepad/recursos.qrc">
    <normaloff>:/recursos/icono.png</normaloff>:/recursos/icono.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="animated">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QPlainTextEdit" name="plainTextEdit">
      <property name="acceptDrops">
       <bool>false</bool>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionAbrir"/>
   <addaction name="separator"/>
   <addaction name="actionGuardar"/>
   <addaction name="separator"/>
   <addaction name="actionGuardar_como"/>
   <addaction name="separator"/>
   <addaction name="actionCortar"/>
   <addaction name="separator"/>
   <addaction name="actionCopiar"/>
   <addaction name="separator"/>
   <addaction name="actionPegar"/>
   <addaction name="separator"/>
   <addaction name="actionDeshacer"/>
   <addaction name="separator"/>
   <addaction name="actionRehacer"/>
   <addaction name="separator"/>
   <addaction name="actionSalir"/>
   <addaction name="separator"/>
   <addaction name="separator"/>
   <addaction name="separator"/>
   <addaction name="actionAcercaDe"/>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>554</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArchivo">
    <property name="title">
     <string>Archivo</string>
    </property>
    <addaction name="actionAbrir"/>
    <addaction name="actionGuardar"/>
    <addaction name="actionGuardar_como"/>
    <addaction name="actionAcercaDe"/>
    <addaction name="actionSalir"/>
   </widget>
   <widget class="QMenu" name="menuEditar">
    <property name="title">
     <string>Editar</string>
    </property>
    <addaction name="actionCortar"/>
    <addaction name="actionCopiar"/>
    <addaction name="actionPegar"/>
    <addaction name="actionDeshacer"/>
    <addaction name="actionRehacer"/>
   </widget>
   <addaction name="menuArchivo"/>
   <addaction name="menuEditar"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAbrir">
   <property name="icon">
    <iconset resource="../QtProjects/notepad/recursos.qrc">
     <normaloff>:/recursos/abrir.png</normaloff>:/recursos/abrir.png</iconset>
   </property>
   <property name="text">
    <string>Abrir</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionGuardar">
   <property name="icon">
    <iconset resource="../QtProjects/notepad/recursos.qrc">
     <normaloff>:/recursos/guardar.png</normaloff>:/recursos/guardar.png</iconset>
   </property>
   <property name="text">
    <string>Guardar</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+G</string>
   </property>
  </action>
  <action name="actionGuardar_como">
   <property name="icon">
    <iconset resource="../QtProjects/notepad/recursos.qrc">
     <normaloff>:/recursos/guardarComo.png</normaloff>:/recursos/guardarComo.png</iconset>
   </property>
   <property name="text">
    <string>Guardar como...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actionSalir">
   <property name="icon">
    <iconset resource="../QtProjects/notepad/recursos.qrc">
     <normaloff>:/recursos/salir.png</normaloff>:/recursos/salir.png</iconset>
   </property>
   <property name="text">
    <string>Salir</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionCortar">
   <property name="icon">
    <iconset resource="../QtProjects/notepad/recursos.qrc">
     <normaloff>:/recursos/cortar.png</normaloff>:/recursos/cortar.png</iconset>
   </property>
   <property name="text">
    <string>Cortar</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionCopiar">
   <property name="icon">
    <iconset resource="../QtProjects/notepad/recursos.qrc">
     <normaloff>:/recursos/copiar.png</normaloff>:/recursos/copiar.png</iconset>
   </property>
   <property name="text">
    <string>Copiar</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionPegar">
   <property name="icon">
    <iconset resource="../QtProjects/notepad/recursos.qrc">
     <normaloff>:/recursos/pegar.png</normaloff>:/recursos/pegar.png</iconset>
   </property>
   <property name="text">
    <string>Pegar</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionDeshacer">
   <property name="text">
    <string>Deshacer</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+U</string>
   </property>
  </action>
  <action name="actionRehacer">
   <property name="text">
    <string>Rehacer</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionAcercaDe">
   <property name="text">
    <string>Acerca de...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+H</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../QtProjects/notepad/recursos.qrc"/>
 </resources>
 <connections/>
</ui>
